src/odemis/dataio/test/hdf5_test.py

```diff
@@ -1126,7 +1126,7 @@ def testReadMDOutWlBands(self):
                      model.MD_IN_WL: (500e-9, 520e-9),  # m
                      model.MD_OUT_WL: ((630e-9, 660e-9), (675e-9, 690e-9)),  # m
                      model.MD_USER_TINT: (255, 0, 65),  # purple
-                     model.MD_LIGHT_POWER: 100e-3  # W
+                     model.MD_LIGHT_POWER: [100e-3]  # W
```

Comment:
No. MD_LIGHT_POWER shouldn't change of format. It's still a float. So please revert these changes in all the dataio module.
In order to make sure it stays the same, you will have to adjust odemisd.mdupdater. Now MD_LIGHT_POWER is slightly more complex to compute: it's not the `power.value`, but the `sum(power.value)`.